blockchain/vm/evm_test.go

```diff
@@ -63,7 +64,7 @@ var PrecompiledContractAddressMappingTestData = []struct {
 	{"0x3fe", feePayerInput, false, Block5, 0, "", kerrors.ErrPrecompiledContractAddress},
 	{"0x3ff", validateInput, false, Block5, 0, "", kerrors.ErrPrecompiledContractAddress},
 	// Condition 2. Caller Contract Deploy - after IstanbulCompatible Change, Call - after istanbulCompatible
-	{"0x009", vmLogInput, true, Block5, 0, "", kerrors.ErrPrecompiledContractAddress},
+	{"0x009", vmLogInput, true, Block5, math.MaxUint64, "", errors.New("invalid input length")},
```

Comment:
You want to test the difference of exactGas from the conditions above, right?

Reply:
Yes, btw this tc's main purpose is to check that the 0x09 precompiled contract runs `blake2f`, not `vmLog` :)

Reply:
Thank you for your explanation.
I have one more question..
If there are multiple purposes in a test, shouldn't the test be divided?